README.md

```diff
@@ -1,6 +1,6 @@
 # Pascal's Triangle
```

Comment:
No .gitignore file and you therefore commit our .DS_Store files. Every project should have a .gitignore file. In addition, look into setting a global gitignore file on your laptop where you can always ignore the .DS_Store files.